.gitignore

```diff
@@ -1,3 +1,5 @@
 build
 .idea
-__pycache__
\ No newline at end of file
+__pycache__
+docs/code_samples_downloaded/*
+!docs/code_samples_downloaded/.gitkeep
```

Comment:
The `code_sampled_downloaded` folder needs to be present so that Python can write into it, but we don't want to commit the actual downloaded (cached) files to Git.